adivinanzas.py

```diff
@@ -7,8 +7,7 @@ def leer(nombre_archivo):
     archivo = open(nombre_archivo, "r")
     
     # Ahora tenemos todo el contenido del archivo en la variable "cuerpo".
-    cuerpo = unicode(archivo.read())
-    cuerpo.decode("utf-8")
+    cuerpo = str(archivo.read())
     # unicode() transforma el string en un string con soporte de acentos
```

Comment:
Why leave outdated comments?

Reply:
removing these comments, sorry for that.

Didn't notice comments while reviewing changes after porting